required/latex-lab/testfiles-OR-luatex/footmisc-003.tlg

```diff
@@ -69,7 +69,7 @@ Completed box being shipped out [1]
 .....\penalty 10000
 .....\pdfliteral page <lua data reference ...>
 .....\pdfliteral page <lua data reference ...>
-.....\pdfstartlink(*+*)x* attr{/Contents (ref) /Border [0 0 1] /H /I /C [0.701176 0.4 0.414118] /StructParent 2 } action struct name{footnote*.5} goto name{footnote*.5}
+.....\pdfstartlink(*+*)x* attr{/Contents <FEFF0047006F00200074006F002000640065007300740069006E006100740069006F006E00200066006F006F0074006E006F00740065002A002E0035> /Border [0 0 1] /H /I /C [0.701176 0.4 0.414118] /StructParent 2 } action struct name{footnote*.5} goto name{footnote*.5}
```

Comment:
why does this PR changes all kind of tlgs? is that pdfmanagement change that shows through hre?

Reply:
yes. I extended/corrected/adjusted the handling of internal links and so now the `/Contents` key has a real content and not only a generic `ref`.